src/Stratis.Bitcoin.Features.Wallet/WalletSyncManager.cs

```diff
@@ -95,8 +98,13 @@ public void Stop()
             this.logger.LogTrace("(-)");
         }
 
-        /// <inheritdoc />
-        public virtual void ProcessBlock(Block block)
+        /// <summary>Called when a block is added to the blocksQueue.
+        /// Depending on the WalletTip and incoming block height, this method will decide whether the block will be processed by the <see cref="WalletManager"/>.
+        /// </summary>
+        /// <param name="block">Block to be processed.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        /// <returns><placeholder>A <see cref="Task"/> representing the asynchronous operation.</placeholder></returns>
```

Comment:
We don't really use returns section for methods that the returns plain task

Reply:
I've removed.

Maybe we need to remove this from Style Cop.  If there return value isn't documented there is a warning :

![image](https://user-images.githubusercontent.com/16000683/43991957-aa579bc6-9d6e-11e8-9823-d66d12901270.png)

Reply:
Yes, it should not produce warning in case there is no return value except for the task